test/dummy_test.js

```diff
@@ -0,0 +1,3 @@
+export function dummy_test(done) {
```

Comment:
Can we add `mocha` to the dependencies and use this for the tests? All new components and tests should be written using mocha.